spec/factories/prior_authority_applications.rb

```diff
@@ -12,6 +12,12 @@
     initialize_with { attributes }
     laa_reference { 'LAA-123456' }
     prison_law { false }
+    provider do
+      {
+        'email' => 'provider@example.com',
+        'description' => nil, # TODO: is this a thing?
```

Comment:
It looks like the provider app sends over a description, which is a value it's received from Portal. But on UAT the description we've received for the test provider is nil, so it might always be nil. 🤷